src/dms-preview/azext_dms/custom.py

```diff
@@ -69,90 +82,85 @@ def create_or_update_project(
                                    group_name=resource_group_name,
                                    service_name=service_name,
                                    project_name=project_name)
+
+
+def get_project_platforms(cmd, project_name, service_name, resource_group_name):
```

Comment:
Can you move get_project_platforms down to where the Help methods region is?

Reply:
Done.